README.md

```diff
@@ -6,8 +6,8 @@ Validators can add their identity, contact information and validator list to the
 
 To add your entity, submit a PR with the following changes:
 
-- **Thumbnail SVG:** Add your entity's thumbnail as an SVG file to [this folder](https://github.com/paritytech/polkadot-staking-dashboard/tree/master/src/config/validators/thumbnails).
-- **Entity details:** Add your entity details to the `VALIDATORS_COMMUNITY`JSON object in [this file](https://github.com/paritytech/polkadot-staking-dashboard/blob/master/src/config/validators/index.ts).
+- **Thumbnail SVG:** Add your entity's thumbnail as an SVG file to [this folder](https://github.com/Cerebellum-Network/staking-dashboard/tree/dev-cere/src/config/validators/thumbnails).
+- **Entity details:** Add your entity details to the `VALIDATORS_COMMUNITY`JSON object in [this file](https://github.com/Cerebellum-Network/staking-dashboard/blob/dev-cere/src/config/validators/index.ts).
```

Comment:
Are you sure we need these links and SVGs?

Reply:
We currently do not have the Cere icon, but it will appear after merging this pull request. As for the other icons, I have decided not to remove them to avoid potential conflicts in the future after merging with the base repository.